server/routes/utils/authentication.js

```diff
@@ -13,8 +13,7 @@ const authentication = (req, res, next) => {
     if (authCheck) {
       next();
     } else {
-      const port = req.socket.localPort && req.socket.localPort !== '80' ? ':' + req.socket.localPort : '';
-      const returnUrl = `${req.protocol}://${req.hostname}${port}${req.originalUrl}`;
+      const returnUrl = process.env.DEV_HUB_URL;
```

Comment:
Do we want to redirect to the URL anyway?

Reply:
Hi Jonathon, there is nothing in the ticket about where to redirect after login. So originally, the return url was being set to the page the user was trying to access. Now the return url is the dev hub homepage.

Reply:
Resolved in conversation with PO.